### DISCUSSION TITLE
Ignore accents in search

### DISCUSSION THREAD

**Discussion by agentydragon** [athensresearch/athens#1384] [Feature Requests]
I am Czech. I have pages which have names which include Czech diacritics, particularly names of people. I'd like to be able to quickly link to such pages without having to switch to a Czech keyboard. Athena (Athens search), and the search pop-up for page links and block references, should match accent-insensitively - for example, a search of `rehor` should also match `Řehoř`.

For example, say an English-speaking user of Athens has a page `Antonín Dvořák` about the Czech composer, and they want to type something like "listened to some music by Dvorak this evening" and link it to the page. It would be good if they could just start typing `[[Dvorak` and get `Antonín Dvořák` autosuggested. Having to type these accented letters is kinda meh - you have to switch keyboards.

This is sort of remotely related to aliases, but it's a simpler feature to implement.

**Comment by Limezy** [comment]
I'm French and need the exact same thing 👍

**Comment by BradKML** [comment]
This is also a good idea, but for some library reference
- https://github.com/codebox/homoglyph
- https://github.com/DavidJacobson/SafeText
- https://github.com/vhf/confusable_homoglyphs
- https://github.com/life4/homoglyphs
- https://github.com/NebulousLabs/glyphcheck
